R/helper-functions.R

```diff
@@ -362,9 +363,9 @@ createTreeViz <- function(clusters, counts) {
 
 #' Finds top variable genes `TreeViz`
 #'
-#' Finds the n top variable genes within the genes present in Summarized Experiment object
+#' Finds the n top variable genes within the genes present in `TreeViz` object
 #'
-#' @param treeseobject TreeViz object
+#' @param treeVizobject TreeViz object
```

Comment:
should only be `treeviz` not `treeVizobject`

Reply:
okay